rest-service/manager_rest/test/test_modify.py

```diff
@@ -49,8 +50,15 @@ def test_modify_add_instance(self):
 
         new_instance = new_instances[0]
         self.assertEqual('node1', new_instance.node_id)
+        expected_old_instances = copy.deepcopy(node_instances1)
+        for instance in expected_old_instances:
+            if instance.node_id == 'node2':
+                current_relationship = instance.relationships[0]
+                new_relationship = copy.deepcopy(current_relationship)
+                new_relationship['target_id'] = new_instance.id
+                instance.relationships.append(new_relationship)
         self.assertEqual(sorted(old_instances, key=lambda _i: _i.id),
-                         sorted(node_instances1, key=lambda _i: _i.id))
+                         sorted(expected_old_instances, key=lambda _i: _i.id))
```

Comment:
Just to make sure, this asserts that both old relationships and added ones are there right?